### ISSUE TITLE
Search for Projects brings up projects connected to creator of project, not just participants

### ISSUE THREAD

**Issue by cjconroy** [ArctosDB/arctos#3915]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Describe the bug**
The main search for projects/pubs interface allows searching by agent name in box for Participant. However, it currently also finds projects that the agent name created, or edited. This is totally superfluous and confusing. 

**To Reproduce**
1. Go to main search page. 
2. Type Conroy into project Participant
3. Results show 28 projects. 15 of these Conroy (me) either created or edited in the last few months. I don't think this was the intended use for this box. The popup for the word Participant reads "Agent who had a role in the project or publication."

**Expected behavior**
Only projects that have name Conroy (for example) in a role like PI, co-PI, student, etc. should come up. 

**Priority**
Moderate. Hopefully easy fix.

**Comment by campmlc** [comment]
Agree. These projects also show up under agent activity. Only formally
added agents should show project relationships. We can have a search tool
for "project edited by" as a separate function.

**Comment by dustymc** [comment]
https://github.com/ArctosDB/arctos/issues/3468 but <img width="135" alt="Screen Shot 2021-09-09 at 3 53 13 PM" src="https://user-images.githubusercontent.com/5720791/132772914-398cc878-5e07-4d1d-bf7e-865a2d476d9a.png"> so ???

**Comment by cjconroy** [comment]
Confusing that there are options here for Participant, and Agent. Participant should be a limited list not including the creator or editor of the Project. "Agent who had a role in the project or publication.". That could be clarified too.  One option to clarify is to use the SAME WORDS. Box for Participant, and drop down for Participant Role.

**Comment by dustymc** [comment]
I agree - that's a bad label (it ain't my fault!), there's nothing in the data or authority that could separate "Participant" from any other agent.

Suggest that converts to a multi-select with some options pre-selected (which puts the distinction, whatever it is, in the UI and makes it obvious) - but that's less likely to get lost in https://github.com/ArctosDB/arctos/issues/3468

**Comment by cjconroy** [comment]
There seems to be something new in the database this year. See screen shots. I am pretty sure I created both of these projects, but my name is not associated with the older one. Could whatever this is doing be used to differentiate participants vs. non participants? The Steppan project does not come up when I query Conroy as participant. But I am just as much a participant as I am with the newer project. Was my role of creator/editor deleted?
<img width="546" alt="Screen Shot 2021-09-09 at 4 25 50 PM" src="https://user-images.githubusercontent.com/5749672/132775312-470f8c7f-1d9f-4d19-b198-4b181f843639.png">
<img width="373" alt="Screen Shot 2021-09-09 at 4 22 24 PM" src="https://user-images.githubusercontent.com/5749672/132775316-3c9864e0-da72-4386-b45e-cbd9deced0f6.png">

**Comment by Jegelewicz** [comment]
>  These projects also show up under agent activity. Only formally added agents should show project relationships. We can have a search tool for "project edited by" as a separate function.

I think these projects SHOULD show up under agent activity, but with a different role:

Project Creator

and this should be part of an agent's Arctos Activity

Then for @cjconroy this role could be eliminated when searching for Project Participants.

We want to understand how many projects someone set up during a given year (part of their curatorial/collection management productivity), but not conflate that with the actual work described in the project which may not pertain to the project creator at all.

**Comment by cjconroy** [comment]
There still seems to be something missing. There are clearly projects that I created showing up, but there are projects I created that are NOT showing up, even when searching generally, Conroy, any role. See projects with Steppan as the agent. Why is that?

**Comment by campmlc** [comment]
Is this related to when the project was created, e.g. before Dusty put in
tracking for creator/editor? Perhaps only newly created projects get
associated with creator?

**Comment by dustymc** [comment]
>different role: Project Creator

That's not what we agreed on, the recent trend of constantly jumping around to new equally-arbitrary values is not sustainable.  https://github.com/ArctosDB/arctos/issues/2769

>Why is that?

https://github.com/ArctosDB/arctos/issues/2570#issuecomment-856829093

The concept is new.

**Comment by Jegelewicz** [comment]
>     different role: Project Creator
> 
> That's not what we agreed on, the recent trend of constantly jumping around to new equally-arbitrary values is not sustainable. #2769

I don't even see what we "agreed" on in that issue except maybe this - https://github.com/ArctosDB/arctos/issues/2570#issuecomment-608045797

> AWG suggestion
> 
>     - make 100-character description hard requirement
>     - autopopulate created by, last edited by - "require" them in the forms to the extent possible
>     - eventually rebuild UI to sorta-maybe-hide admin agent roles

and it seems like we are doing the second bullet, but not the third? This issue is about the third.

**Comment by krgomez** [comment]
I just created a bunch of projects associated with exhibitions. I have now realized that all of these projects show up on my agent profile even though I had nothing to do with them other than I entered this legacy information into Arctos. Is there anyway to change the creator of a project to another agent if this relationship to projects causes them to be prominently displayed on an agent summary? I think it is misleading to have my name associated with these projects on my profile in Arctos.

**Comment by Jegelewicz** [comment]
Instead of "creator" as a label for the person who made the project in Arctos, perhaps we should use "entered by"?

**Comment by Jegelewicz** [comment]
HAHAHAHAHAHA!!!! Thanks for that.....

BUT I do see @krgomez point on her agent page (although I believe she SHOULD get credit for all of that work). When looking at an agent page you can't see what role a person played in the project. Maybe we need that there somehow? @krgomez I WANT all the projects I created on my agent page because I took the time to create them. Had I not done so, the work would probably still be "in the dark" for everyone involved - I think that "publicist" is a pretty important role....

**Comment by campmlc** [comment]
No one can ever say consistency is our strong point. Maybe we should be able to choose which projects are displayed publicly in our agent page, like Research gate and LinkedIn?

**Comment by ewommack** [comment]
> Had I not done so, the work would probably still be "in the dark" for everyone involved - I think that "publicist" is a pretty important role....

I like this point by @Jegelewicz. If I have volunteer or student running through and creating projects as their intern project, I would definitely want them to get credit. I'd like credit for those projects that I create, especially the ones I put media up in, or work to make it into something the researcher will use to highlight their research.

**Comment by campmlc** [comment]
Yes, but we need a separate display in our agent page. Right now I'm an Academy of Science level researcher based on the projects I've "participated in" on my agent page, but most are just projects I created to document loans.
